tests/test_pytumblr.py

```diff
@@ -20,318 +30,309 @@ def setUp(self):
+        mock_get.side_effect = wrap_response('{"meta": {"status": 200, "msg": "OK"}, "response": {"liked_posts": [] } }')
 
         response = self.client.blog_likes('codingjester.tumblr.com', after=1418684291)
         assert response['liked_posts'] == []
 
-    @httprettified
-    def test_blogLikes_with_before(self):
-        HTTPretty.register_uri(HTTPretty.GET, 'https://api.tumblr.com/v2/blog/codingjester.tumblr.com/likes',
-                               body='{"meta": {"status": 200, "msg": "OK"}, "response": {"liked_posts": [] } }')
+    @mock.patch('requests.get')
+    def test_blogLikes_with_before(self, mock_get):
+        mock_get.side_effect = wrap_response('{"meta": {"status": 200, "msg": "OK"}, "response": {"liked_posts": [] } }')
 
         response = self.client.blog_likes('codingjester.tumblr.com', before=1418684291)
         assert response['liked_posts'] == []
 
-    @httprettified
-    def test_queue(self):
-        HTTPretty.register_uri(HTTPretty.GET, 'https://api.tumblr.com/v2/blog/codingjester.tumblr.com/posts/queue',
-                               body='{"meta": {"status": 200, "msg": "OK"}, "response": {"posts": [] } }')
+    @mock.patch('requests.get')
+    def test_queue(self, mock_get):
+        mock_get.side_effect = wrap_response('{"meta": {"status": 200, "msg": "OK"}, "response": {"posts": [] } }')
 
         response = self.client.queue('codingjester.tumblr.com')
         assert response['posts'] == []
 
-    @httprettified
-    def test_drafts(self):
-        HTTPretty.register_uri(HTTPretty.GET, 'https://api.tumblr.com/v2/blog/codingjester.tumblr.com/posts/draft',
-                               body='{"meta": {"status": 200, "msg": "OK"}, "response": {"posts": [] } }')
+    @mock.patch('requests.get')
+    def test_drafts(self, mock_get):
+        mock_get.side_effect = wrap_response('{"meta": {"status": 200, "msg": "OK"}, "response": {"posts": [] } }')
 
         response = self.client.drafts('codingjester.tumblr.com')
         assert response['posts'] == []
 
-    @httprettified
-    def test_submissions(self):
-        HTTPretty.register_uri(HTTPretty.GET, 'https://api.tumblr.com/v2/blog/codingjester.tumblr.com/posts/submission',
-                               body='{"meta": {"status": 200, "msg": "OK"}, "response": {"posts": [] } }')
+    @mock.patch('requests.get')
+    def test_submissions(self, mock_get):
+        mock_get.side_effect = wrap_response('{"meta": {"status": 200, "msg": "OK"}, "response": {"posts": [] } }')
 
         response = self.client.submission('codingjester.tumblr.com')
         assert response['posts'] == []
 
-    @httprettified
-    def test_follow(self):
-        HTTPretty.register_uri(HTTPretty.POST, 'https://api.tumblr.com/v2/user/follow',
-                               body='{"meta": {"status": 200, "msg": "OK"}, "response": []}')
+    @mock.patch('requests.post')
+    def test_follow(self, mock_post):
+        def mocked_post(*args, **kwargs):
```

Comment:
Should we have something like `wrap_response` for POST requests as well to reduce duplication?

Reply:
👌 sounds good